Assignments/load_baseball_data.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+pwd
+
+sleep 30
+
+#Got some help from Will but wasnt able to figure it out.
+if ! mysql -h mariadb -u root -pBDAMaster -e 'use baseball'; then
+  echo "Loading baseball data into SQL"
```

Comment:
adding these echos in the bash file was a good idea. I wish I'd done that but going back to change mine at this late hour seems risky.